AzureAD/Remove-AadAppsForBc.ps1

```diff
@@ -47,16 +54,19 @@ try {
             if ($jwtToken.aud -ne 'https://graph.microsoft.com') {
                 Write-Host -ForegroundColor Yellow "The accesstoken was provided for $($jwtToken.aud), should have been for https://graph.microsoft.com"
             }
-            Connect-MgGraph -AccessToken $bcAuthContext.accessToken
+            $accessToken = $bcAuthContext.accessToken
         }
-        else {
-            if ($accessToken) {
-                Connect-MgGraph -accessToken $accessToken
+        if ($accessToken) {
+            if ($useSecureStringForAccessToken){
+                Connect-MgGraph -AccessToken (ConvertTo-SecureString -String $accessToken -AsPlainText -Force) | Out-Null
             }
             else {
-                Connect-MgGraph
+                Connect-MgGraph -AccessToken $accessToken | Out-Null
             }
         }
+        else {
+            Connect-MgGraph -Scopes 'Application.ReadWrite.All' | Out-Null

```

Comment:
This comes into play when you have not specified useCurrentMicrosoftGraphConnection and not specified a bcAuthContext nor an accesstoken.
What does it do?

Reply:
This tries to connect with Microsoft Graph via UI Interaction/Login via Browser and requests this scope via delegated user login. 
I only copied this from the the other cmdlet `New-AadAppsForBc`